internal/commands/set_registry.go

```diff
@@ -0,0 +1,54 @@
+package commands
+
+import (
+	"github.com/spf13/cobra"
+	"gopkg.in/errgo.v2/fmt/errors"
+
+	"github.com/buildpacks/pack/internal/config"
+	"github.com/buildpacks/pack/internal/style"
+	"github.com/buildpacks/pack/logging"
+)
+
+func SetRegistry(logger logging.Logger, cfg config.Config, cfgPath string) *cobra.Command {
+	var (
+		registryName string
+	)
+
+	cmd := &cobra.Command{
+		Use:   "set-registry <name>",
+		Args:  cobra.ExactArgs(1),
+		Short: "Set default registry",
+		RunE: logError(logger, func(cmd *cobra.Command, args []string) error {
+			registryName = args[0]
+			if !containsRegistry(config.GetRegistries(cfg), registryName) {
+				return errors.Newf("no registry with the name %s exists", style.Symbol(registryName))
```

Comment:
I don't think we should be importing yet another error lib for a single function.

`pkg/errors` provides this functionality via `errors.Errorf`.

Reply:
Agreed.  I must have accidentally done this.